installation/check_requirements.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+
+function check_version {
+  local package="$1"
+  local cmd="$2"
+  local expected_version="$3"
+  local version=$cmd
+ 
+  IFS='.' read -ra version_arr <<< "$version"
+  local major=${version_arr[0]#v}
+  local minor=${version_arr[1]}
+  
+  IFS='.' read -ra e_version_arr <<< "$expected_version"
+  local expected_major=${e_version_arr[0]}
+  local expected_minor=${e_version_arr[1]}
+  
+  if [ "$major" -lt "$expected_major" ] || [ "$minor" -lt "$expected_minor" ]; then
+    echo "Warning: $package version is incorrect. Expected $expected_version but got $version"
+  else
+    echo "$package version is correct: $version"
```

Comment:
I got this error:
<img width="692" alt="image" src="https://user-images.githubusercontent.com/34718875/221321731-3d74f0eb-5f34-41b7-8187-591f6708e641.png">

Reply:
The command worked fine for me, although I use `zsh` instead of `bash`.

Reply:
Seems that for some reason the script didn't found those packages 🤔